### PR TITLE
added norwegian translation

### PR DESCRIPTION
**Pull request by skandix**
Signed-off-by: Bendik Egenes Dyrli <bendik.dyrli@uia.no>

### CODE REVIEW

**Comment by TheLastProject** [comment]
I had to move it from `values-no` to `values-nb` to have it show up on Android. Maybe I don't understand Norwegian well? But unless you know better it seems like the directory should be renamed.

My language settings for testing were as followed:
image

App looks like this when translated:
image

**Comment by skandix** [comment]
that's true it should be value-nb, i'll fix and I see something i need to rephrase :) :+1:

**Comment by TheLastProject** [comment]
There's new stuff to translate, take a look at https://github.com/TheLastProject/RaiseToAnswer/commit/4606d718889f030c83bb3df3d9fad9957bab2e85#diff-01dafb3fd0217441330103cfc8300094 to see what changed.

**Comment by skandix** [comment]
> There's new stuff to translate, take a look at [4606d71#diff-01dafb3fd0217441330103cfc8300094](https://github.com/TheLastProject/RaiseToAnswer/commit/4606d718889f030c83bb3df3d9fad9957bab2e85#diff-01dafb3fd0217441330103cfc8300094) to see what changed.

\o had some time off , but finishing this now :)

File: app/src/main/res/values-nb/strings.xml [hunk 1]
```diff
@@ -1,24 +1,38 @@
...
+
+    <string name="answer_holder">Answering calls</string>
+
+    <string name="answer_all_angles_enabled_key" translatable="false">answer_all_angles_enabled_key</string>
+
+    <string name="missing_support_magnetometer">Your device lacks a magnetometer, declining or limiting answering to an up-right angle is not supported.</string>
+
+    <string name="missing_support_android_9">Your device is running a version of Android before 9, declining calls is not supported.</string>
+</resources>
+</resources>
```
Comment (TheLastProject) [inline app/src/main/res/values-nb/strings.xml:38]: Can you remove this duplicate </resources> tag?

Comment (skandix) [reply app/src/main/res/values-nb/strings.xml:38]: ah yes :)

**Pull request merged**